### ISSUE TITLE
Safari bitwarden extension not accessible

### ISSUE THREAD

**Issue by hfath** [bitwarden/browser#1211]
My setup:

Mac OS X 10.12.6
Safari 12.1.2
Bitwarden desktop client 1.17.2
Bitwarden Browser extension 1.43.4

The bitwarden extension shows up in the Settings::Extensions overview, but I do not get a window bar icon (and have not for a while), i.e. the extension is non-functional.

Older bitwarden versions up to 1.16.6 appear to crash Safari with "BUG IN CLIENT OF LIBDISPATCH: trying to lock recursively" (stack traces upon request); 1.17.2 hasn't (yet) done that, and I haven't found any logs from it.

**Comment by hfath** [comment]
There are, in fact, related crashes:


[safari_2020-04-21-115744_MacBook-Pro_crash.txt](https://github.com/bitwarden/browser/files/4509192/safari_2020-04-21-115744_MacBook-Pro_crash.txt)

**Comment by nealeyoung** [comment]
Same issue (bitwarden icon is not showing in the Safari toolbar).  For me disabling and re-enabling the extension brings it back.  I have this problem whether or not the bitwarden app is running too.

**Comment by hfath** [comment]
Further observation: Both after starting Safari while the BitWarden desktop application is open, and after toggling the BitWarden checkmark in Safari Settings, the BitWarden icon will show up to the left of the Safari URL bar for a split second, then disappear.

The console log has
<pre>
[...]
standard	09:45:02.449714 +0200	Safari	completed calling plugIn beginUsing: for pid: 4070
standard	09:45:02.463236 +0200	Safari	Unable to setup extension context - error: Die Kommunikation mit einem Hilfsprogramm ist fehlgeschlagen.
standard	09:45:02.463368 +0200	Safari	connection to plug-in &lt;private&gt;(FD1DA946-96CF-48FB-9A1F-37AF24C588ED) lost
standard	09:45:02.463408 +0200	Safari	extension connection was interrupted
fehler	09:45:02.463539 +0200	Safari	Connecting to the extension with identifier &lt;private&gt; failed with error Error Domain=NSCocoaErrorDomain Code=4097 "Die Kommunikation mit einem Hilfsprogramm ist fehlgeschlagen."
standard	09:45:02.463954 +0200	Safari	connection to plug-in &lt;private&gt;(&lt;private&gt;) lost
fehler	09:45:02.474458 +0200	ReportCrash	Failed to start process notifications for pid 4070 (16)
standard	09:45:02.821576 +0200	ReportCrash	Saved crash report for &lt;private&gt; to &lt;private&gt;
[...]
</pre>

**Comment by hfath** [comment]
(MacOS version 10.12.6, Safari 12.1.12, BitWarden 20.1)

**Comment by lvtz** [comment]
Same here:

In an undefined situation, the icon disappears from the Safari toolbar. Re-enabling via Extensions tab in Safari settings resolving temporary this issue. Faced this issue months ago, but issue is more annoying to me in last days... so I decided to report it.

Issue occurs on Safari with Catalina 10.5.7 and Big Sur 11.0.1. Safari Version 14.0.1 (16610.2.11.51.8). Bitwarden App was updated to latest version downloaded via AppStore.
